src/buffer/out/textBuffer.hpp

```diff
@@ -141,7 +151,7 @@ class TextBuffer final
     bool MoveToNextGlyph(til::point& pos, bool allowBottomExclusive = false) const;
     bool MoveToPreviousGlyph(til::point& pos) const;
 
-    const std::vector<SMALL_RECT> GetTextRects(COORD start, COORD end, bool blockSelection = false) const;
+    const std::vector<SMALL_RECT> GetTextRects(COORD start, COORD end, bool blockSelection = false, bool bufferCoordinates = false) const;

```

Comment:
Outside of the unit tests, it looks like there are only two that are false (and three that are true). Would it perhaps be better to get rid of the default value, and always just set the `bufferCoordinates` parameter explicitly? We can get rid of the `blockSelection` default too, since that's already always being set explicitly (except maybe some of the unit tests, but I think there are only a handful of those).

Reply:
Sure. I like making it explicit and cleaning that up.